### PR TITLE
1 site header

### PR DESCRIPTION
**Pull request by alisherj0n**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,6 +13,46 @@
...
+            <li>
+              <a href="#" class="link"
+                ><span data-hover="Контакты">Контакты</span></a
+              >
+            </li>
+          </nav>
+
+          <button class="phone">
+            <a href="tel:8(495)1424043" class="number">8 (495) 142 40 43</a>
+          </button>
```
Comment (AzamatRasulov) [inline index.html:53]: Link cannot be inside a button.

File: src/_base.scss [hunk 2]
```diff
@@ -1,12 +1,57 @@
...
 *::after {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
 }
+
+body {
+  font-family: Mulish, system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI',
+    Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  height: 300vh;
```
Comment (AzamatRasulov) [inline src/_base.scss:12]: ?

File: src/_base.scss [hunk 3]
```diff
@@ -1,12 +1,57 @@
...
+    transform: rotateX(-90deg);
+    transform-origin: 50% 0;
+    text-align: center;
+  }
+
+  &:hover span,
+  &:focus span {
+    transform: rotateX(90deg) translateY(-22px);
+  }
+}
```
Comment (AzamatRasulov) [inline src/_base.scss:57]: I'd extract it into a component

File: src/_layout.scss [hunk 4]
```diff
@@ -1,7 +1,7 @@
 .container {
   max-width: 75rem;
   margin: auto;
-  padding: 2rem 1rem;
+  padding: 3rem 1rem;
```
Comment (AzamatRasulov) [inline src/_layout.scss:4]: Than's certainly too much. Check it on smaller screen. If you want your container to be narrower, there's a dedicated property for ~~width бо намефами~~ max-width.

File: src/components/_header.scss [hunk 5]
```diff
@@ -0,0 +1,44 @@
+.site-header {
+  background-color: transparent;
```
Comment (AzamatRasulov) [inline src/components/_header.scss:2]: What does this do?

File: index.html [hunk 6]
```diff
@@ -13,6 +13,46 @@
...
   </head>
   <body>
-    <h1>Hello World</h1>
+    <header class="site-header">
+      <div class="container content">
+        <a href="/" class="site-logo">
+          <img src="./images/logo.png" alt="red and blue logo" class="logo" />
+        </a>
+
+        <div class="navigation">
```
Comment (AzamatRasulov) [inline index.html:22]: You don't need this element

File: src/components/_header.scss [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+    .navigation {
+      display: flex;
+      align-items: center;
+      gap: 2rem;
+
+      * {
+        flex-shrink: 0;
+      }
+      nav {
```
Comment (AzamatRasulov) [inline src/components/_header.scss:25]: no selection by tag

**Review by AzamatRasulov — CHANGES_REQUESTED** [review]

File: src/_base.scss (hunk 2 above)
Comment (alisherj0n) [reply src/_base.scss:12]: It was needed for sticky test

**Review by AzamatRasulov — APPROVED** [review]

**Pull request merged**